doc/cephfs/index.rst

```diff
@@ -1,8 +1,10 @@
+.. _ceph-filesystem:
+
 =================
  Ceph Filesystem
 =================
 
-The :term:`Ceph Filesystem` (Ceph FS) is a POSIX-compliant filesystem that uses
+The Ceph Filesystem (Ceph FS) is a POSIX-compliant filesystem that uses
```

Comment:
While I'm fine with the Glossary having both "CephFS" and "Ceph FS", let's just use "CephFS" everywhere else.

Reply:
@batrick 

* Fixed in `Ceph FS` in index.rst. But there are other places to clean up.
* I have added a url to `Architecture - Ceph Filesystem` in the glossary, as it has a good explanation.

Please check again.